<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c_ylxmcsdb</class>
 <widget class="QDockWidget" name="c_ylxmcsdb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医疗项目次数对比</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,7">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,1,0,1,0,1,1">
        <item row="1" column="0">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="1" column="6">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>开始日期</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>全部</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>科室</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>结束日期</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>项目名称（关键字查询）</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QDateEdit" name="dateEdit_2"/>
        </item>
        <item row="0" column="7">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,1">
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string> 重置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>月度对比</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <column>
             <property name="text">
              <string>科室</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>项目类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>项目名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>去年同期例数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>今年例数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>例数相差</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>季度对比</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget_2">
            <column>
             <property name="text">
              <string>科室</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>项目类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>项目名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>去年同期例数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>今年例数</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>例数相差</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
